kaiax/reward/impl/getter.go

```diff
@@ -427,12 +428,12 @@ func specWithProposerAndFunds(spec *reward.RewardSpec, config *reward.RewardConf
 	}
 
 	newSpec.Proposer = proposer
-	if si.CLStakingInfos == nil {
+	if !config.Rules.IsPrague {
```

Comment:
`CLStakingInfo` can be nil after Prague

```suggestion
	if !config.Rules.IsPrague || si.CLStakingInfos == nil {
```

Reply:
Thanks, reflected at [5fe6901](https://github.com/kaiachain/kaia/pull/193/commits/5fe690147f28fd308edd881cda93e73ee08b8e96).